modules/custom/az_core/src/Commands/README.md

```diff
@@ -0,0 +1,31 @@
+# AZ Core Config Commands
+
+A Drupal module that provides custom Drush commands for managing the configuration of the AZ Quickstart profile. The module adds missing permissions from the AZ Quickstart profile to the active site.
+
+## Requirements
+- Drupal 8 or 9
+- Drush
+
+## Installation
+- Install the module as you would normally install any Drupal module.
+- Enable the module using the Drupal administration interface or Drush.
+- After installation, you can access the custom Drush commands provided by this module.
```

Comment:
Actually, this part is likely not correct, since az_core is for az_quickstart, and not meant for generic usage.

Reply:
Assuming that az_core is a dependency of az_quickstart and enabled by default, I could update this to say:

```suggestion
## Installation
This module is automatically included in the az_quickstart install profile and enabled by default. This custom Drush command will work on any installed Quickstart 2 website.
```